uint32 robotID
---
sensor_msgs/LaserScan laserScan
